lib/huiClient.js

```diff
@@ -0,0 +1,47 @@
+const log = require('@financial-times/n-logger').default;
+
+const huiBasePath = process.env.HUI_BASE_PATH;
+const licencePath = '/licences/';
+const options = {
+  method: 'GET',
+  headers: {
+    'X-Api-Key': process.env.HUI_API_KEY
+  }
+};
+
+function getUsage(licenceId, aggregation, startDate, endDate, type = '', filter = '') {
+  let start = '';
+  let end = '';
+  let url = `${huiBasePath}${licencePath}${licenceId}?aggregation=${aggregation}`;
+
+  // Get the prev week/month data too. We make a clone of the start date because
+  // moments are mutable.
+  const adjustedStartDate = startDate.clone().startOf(aggregation === 'week' ? 'isoweek' : aggregation).subtract(1, aggregation);
+
+  const dateFormat = aggregation === 'week' ? '[wk]W/GGGG' : '[m]M/GGGG';
```

Comment:
Yeah, saw that as well. This code uses the ISO Week (the capital `W` in the `dateFormat`) and Year (`GGGG`), as per the MomentJS docs: https://momentjs.com/docs/#/displaying/